powerstore/resource_host_group.go

```diff
@@ -68,7 +69,21 @@ func (r *resourceHostGroup) Schema(ctx context.Context, req resource.SchemaReque
 
 			"host_ids": schema.SetAttribute{
 				ElementType:         types.StringType,
-				Required:            true,
+				Optional:            true,
+				Computed:            true,
+				Description:         "The list of hosts to include in the host group.",
+				MarkdownDescription: "The list of hosts to include in the host group.",
+				Validators: []validator.Set{
+					setvalidator.ConflictsWith(path.Expressions{
+						path.MatchRoot("host_names"),
+					}...),
+				},
+			},
+
+			"host_names": schema.SetAttribute{
+				ElementType:         types.StringType,
+				Optional:            true,
```

Comment:
The list of host names to include in the host group

Reply:
done